doc/source/User_guide/index.rst

```diff
@@ -1,60 +1,73 @@
 .. _ref_user_guide:
 
-==========
-User guide
-==========
-PyAEDT works both inside AEDT and as a standalone application.
-It automatically detects whether it is running in an IronPython (limited capabilities) or CPython
-environment and initializes AEDT accordingly. PyAEDT also provides
-advanced error management.
+=========
+Tutorials
```

Comment:
We on the PyAnsys team want all libraries to use the same section titles where possible. I don't see a reason why this section can't remain named "User guide". I'm tagging @MaxJPRey for feedback.